keyboards/winkeyless/bface/bface.c

```diff
@@ -37,10 +30,59 @@ void rgblight_set(void) {
     }
 
     i2c_init();
-    i2c_send(0xb0, (uint8_t*)led, 3 * RGBLED_NUM);
+    i2c_transmit(0xb0, (uint8_t*)led, 3 * RGBLED_NUM, 100);
+}
+#endif
+
+void matrix_init_kb(void) {
+#ifdef RGBLIGHT_ENABLE
+    if (rgblight_config.enable) {
+        i2c_init();
+        i2c_transmit(0xb0, (uint8_t*)led, 3 * RGBLED_NUM, 100);
+    }
+#endif
+    // call user level keymaps, if any
+    matrix_init_user();
+}
+
+void matrix_scan_kb(void) {
+#ifdef RGBLIGHT_ENABLE
+    rgblight_task();
+#endif
+    matrix_scan_user();
+    /* Nothing else for now. */
 }
 
 __attribute__ ((weak))
 void matrix_scan_user(void) {
-    rgblight_task();
+}
+
+void backlight_init_ports(void) {
+    // initialize pins D0, D1, D4 and D6 as output
+    setPinOutput(D0);
+    setPinOutput(D1);
+    setPinOutput(D4);
+    setPinOutput(D6);
+
+    // turn backlight LEDs on
+    writePinHigh(D0);
+    writePinHigh(D1);
+    writePinHigh(D4);
+    writePinHigh(D6);
+}
+
+void backlight_set(uint8_t level) {
```

Comment:
The Backlight feature supports defining multiple pins in software PWM mode: https://docs.qmk.fm/#/feature_backlight?id=multiple-backlight-pins
Though it is apparently experimental, might be worth a try. Apart from D4, the other three pins here are not hardware PWM capable anyway on the 32A.

Reply:
I really hope people stop making BMC compatible boards. Old boards I can understand but new ones, not so much.